pkg/cmd/kafka/create/create.go

```diff
@@ -93,10 +95,14 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 				}
 			}
 
-			if opts.bypassChecks && (opts.marketplace != "" || opts.marketplaceAcctId != "") {
+			if opts.bypassChecks && (opts.marketplace != "" || opts.marketplaceAcctId != "" || opts.billingModel != "") {
 				return f.Localizer.MustLocalizeError("kafka.create.error.bypassChecks.marketplace")
 			}
 
+			if opts.billingModel == "standard" && (opts.marketplaceAcctId != "" || opts.marketplace != "") {
```

Comment:
I would suggest to not list any billing model in create.go. Also not hardcoding it's value.